client/package.json

```diff
@@ -30,5 +30,5 @@
   "devDependencies": {
     "axios": "^0.19.0"
   },
-  "proxy": "http://localhost:8000"
+  "proxy": "http://localhost:3001"
```

Comment:
keep server proxy at 8000.
on ur server side .env, make sure PORT it's also 8000